WORKSPACE

```diff
@@ -57,6 +57,39 @@ load(
 
 container_repositories()
 
+container_pull(
+    name = "distroless",
+    digest = "sha256:164f3e5a414bffe3350f2fdcf6a2dd2a1fd0051f3ff12ed5d6f07de054fdeaf9",
+    registry = "gcr.io",
+    repository = "distroless/base",
```

Comment:
Are we wrapping pure go binaries or cgo binaries?

Pure go probably wants `distroless/static`; cgo will need `distroless/base`. 
https://github.com/GoogleContainerTools/distroless/blob/master/base/README.md

Reply:
/static works for me. Changed.